src/twisted/internet/test/test_base.py

```diff
@@ -369,3 +369,65 @@ def test_str(self):
             "traceback at creation:".format(id(dc)))
         self.assertRegex(
             str(dc), expectedRegexp)
+
+
+
+class TestSpySignalCapturingReactor(ReactorBase):
+
+    """
+    Subclass of ReactorBase to capture signals delivered to the
+    reactor for inspection.
+    """
+
+    def installWaker(self):
+        """
+        Required method, unused.
+        """
+        pass
```

Comment:
`pass` is not necessary here, the docstring is sufficient.  (Sorry, twistedchecker ought to check for this.)